src/index.html

```diff
@@ -4,10 +4,13 @@
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Leaderboard website</title>
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 <body>
    <main class="container">
-<h1>Leaderboard</h1>
+      
+<h1> <i class="fa-sharp fa-solid fa-trophy"></i>Leaderboard<i class="fa-sharp fa-solid fa-trophy"></i></h1>
+
```

Comment:
- [ ] Please move the "Leaderboard" `<h1>` logo to the left just as in the wireframe shown below 👇 
### Wireframe
![image](https://github.com/MariamOsman11/Leaderboard/assets/46839548/02b44616-6a2b-41bc-a215-1d6773a97051)

### Your Design
![image](https://github.com/MariamOsman11/Leaderboard/assets/46839548/36f07643-3253-481e-8198-b47bd621b108)

- [ ] [OPTIONAL] Please consider changing the font-color of "Leaderboard" `h1` logo because it is currently difficult to read the text and that will bring up accessibility issues 👍